src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)].name();
```

Comment:
It would be more efficient to initialize the 'colors' array outside of the 'getRandomColor' method. This way, the array doesn't have to be reinitialized every time the method is called.

Reply:
done